### ISSUE TITLE
A comprehensive guide to the target traits?

### ISSUE THREAD

**Issue by Lampshade4444** [wyanido/pokebot-nds#60]
pokemon.parse_data is very confusing to dig through as someone who doesn't code. Despite multiple attempts, I have not been able to understand how to use target criteria other than the default (IV sum). I think it would be helpful for many less tech-savy users to provide some actual guide on what commands should be used to look for specific traits in pokemon (Natures, shinyness, specific IVs?) as well as how to input them for "all true" or "any true". I know it's technically listed in the pokemon.parse_data, but it's not comprehensible to me the way it's listed there.

If it exists already, please redirect me. I might be blind,

**Comment by Alexwaukee** [comment]
I had the same though. It could definitely be made more user friendly! 
Replacing the big text box with dropdown menus might be the easiest fix, as to make sure the input is always valid.
Want a specific nature? Click the nature dropdown, select the nature you want. That seems intuitive to me at least.
I could probably come up with a fork and PR something to change the dashboard to be more beginner friendly.

**Comment by CPTMUDD** [comment]
Yeah I can't make heads or tails of what I need to put in there. I did find "local key_whitelist" that contained "ivSum", so I assume the others there such as "spAttackIV" would be put in the same way as the ivSum, but I don't know what an 'array' is or how to input an AND argument

For example I only want it to trigger if the HP, spA, *and* speed IVS are 31
